src/appleseed-max-impl/appleseedrenderer/renderersettings.h

```diff
@@ -55,9 +55,19 @@ class RendererSettings
     // Image Sampling.
     //
 
-    int         m_pixel_samples;
+    int         m_uniform_pixel_samples;
```

Comment:
Move this to a new "Uniform Pixel Sampler" section below, like you did with "Adaptive Tile Sampler".

Reply:
Ok